docs/book/src/plugins/go-v4-plugin.md

```diff
@@ -24,6 +24,8 @@ under the [testdata][testdata] directory on the root directory of the Kubebuilde
 - If you are looking to have your project update with the latest version available
 - if you are not targeting k8s versions < `1.16` and `1.20` if you are using webhooks
 - If you are looking to work on with scaffolds which are compatible with k8s `1.25+`
+- If you are looking for the new layout where to maintain good Go hygiene the "api(s)" and "controller(s)" are scaffold 
```

Comment:
nit: Maybe refer to "Standard Go Project Layout" instead of "good Go hygiene"? I am pretty sure the current layout is not "bad Go hygiene". 😜